Userland/Libraries/LibGfx/ImageFormats/BMPLoader.cpp

```diff
@@ -1429,7 +1429,7 @@ static ErrorOr<void> decode_bmp_pixel_data(BMPLoadingContext& context)
             TRY(process_row(row));
         }
 
-        if (context.is_included_in_ico) {
+        if (context.is_included_in_ico && streamer.remaining() != 0) {
```

Comment:
This is minor, but we have `streamer.at_end()` for this purpose

Reply:
Sure!